second_testapp/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+from typedmodels.models import TypedModel
+
```

Comment:
```suggestion
from testapp.models import Order  # NOQA
```
Probably there is something wrong with Django model discovery order, the test will pass with this line.

Reply:
The whole idea is to avoid circular import dependency. Simply removing this line
```python
                    field.do_related_class = types.MethodType(do_related_class, field)
```

Fixes the issue